DataManagementSystem/Client/DataManager.py

```diff
@@ -16,7 +16,8 @@
 import fnmatch, os, time
 # # from DIRAC
 import DIRAC
-from DIRAC import S_OK, S_ERROR, DError, DErrno, gLogger, gConfig
+from DIRAC import S_OK, S_ERROR, gLogger, gConfig
+from DIRAC.Core.Utilities import DErrno, DError
```

Comment:
No, I made the DError object importable directly from the Utilities module to avoid 2 lines everywhere